### DISCUSSION TITLE
v0.12.1

### DISCUSSION THREAD

**Discussion by vilhelmprytz** [vilhelmprytz/pterodactyl-installer#360] [General]
# v0.12.1

Introducing v0.12.1 - this is a hotfix release that comes with a bug fix. A bug caused the script to fail on some systems because `curl` was not able to overwrite existing `/tmp/lib.sh`. Thanks to @Dany01000110 for reporting!

### Fixed

- [#359](https://github.com/pterodactyl-installer/pterodactyl-installer/issues/359) lib: Fix a bug that would cause the script to fail on some systems because `curl` was not able to overwrite existing `/tmp/lib.sh`.

<hr /><em>This discussion was created from the release <a href='https://github.com/pterodactyl-installer/pterodactyl-installer/releases/tag/v0.12.1'>v0.12.1</a>.</em>

**Comment by MaskiosS** [comment]
Bro, I'm having trouble installing pterodactyl using your script in this new version, always at the end of installing this error.
so the installation always stop, and I can't access the panel.

Created Teamspeak3 Server
Created Mumble Server

****************************************
*     Updating Eggs for Nest: Rust     *
****************************************

Created Rust

  Database\Seeders\EggSeeder ................................. 258.51 ms DONE  


In Model.php line 53:
                                                                              
  Could not save Pterodactyl\Models\User[]: failed to validate data: {"usern  
  ame":["The username must start and end with alpha-numeric characters and\n  
                  contain only letters, numbers, dashes, underscores, and pe  
  riods."]}

**Reply by Linux123123** [reply]
**The username must start and end with alpha-numeric characters and contain only letters, numbers, dashes, underscores, and periods**

**Reply by MaskiosS** [reply]
The error is in the script, since it installs everything automatically, there is no way to choose which egg will be installed, it tries to install the egg rust, automatically. I can not edit, or change anything, then the error ta in the script.

**Reply by Linux123123** [reply]
<h2>NO. </h2> <h3> The problem is your username. The username must start and end with alphanumeric characters and contain only letters, numbers, dashes, underscores, and periods</h3>
The part where it seeds eggs is already done at that point.
